Automattic/jetpack
# Social: Create a social service connect button

This is currently a proof of concept where we get the available external services and render a connect button, which in turn triggers the authentication flow for each service.

To test it, the site must be sandboxed with D146389-code applied.





## Proposed changes:

* Wire up connect buttons for the new connections management UI

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Goto Jetpack and Social admin pages
* Click on Add new connection
* Click on connect for any social network and complete the auth
* Confirm that you see the appropriate data in console
* Do the same for Mastodon
* Confirm that you are take to the next step to enter username
* Enter some wrong username
* Confirm that you see a notice saying the username is invalid
* Enter a valid username and complete the auth
* Confirm that you see the connection data in console
* Now connect Mastodon for the same site via Calypso
* Come back to Social/Jetpack page and reload
* Now try to connect the same Mastodon account
* Confirm that you see the notice that the account is already connected


https://github.com/Automattic/jetpack/assets/18226415/0656e4ce-2be2-4171-aec9-90651c89e7c6
